packages/video_player/video_player_avfoundation/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -236,6 +242,90 @@ - (instancetype)initWithPlayerItem:(AVPlayerItem *)item
   return self;
 }
 
+- (void)setPictureInPicture:(BOOL)pictureInPictureEnabled {
+  if (self.pictureInPictureEnabled == pictureInPictureEnabled) {
+    return;
+  }
+
+  self.pictureInPictureEnabled = pictureInPictureEnabled;
+  if (_pipController && self.pictureInPictureEnabled &&
+      ![_pipController isPictureInPictureActive]) {
+    if (_eventSink != nil) {
+      _eventSink(@{@"event" : @"startingPiP"});
+    }
+    [_pipController startPictureInPicture];
+  } else if (_pipController && !self.pictureInPictureEnabled &&
+             [_pipController isPictureInPictureActive]) {
+    [_pipController stopPictureInPicture];
+  }
+}
+
+#if TARGET_OS_IOS
+- (void)setupPipController:(BOOL)canStartPictureInPictureAutomaticallyFromInline {
+  if ([AVPictureInPictureController isPictureInPictureSupported]) {
+    _pipController = [[AVPictureInPictureController alloc] initWithPlayerLayer:self.playerLayer];
+    if (@available(iOS 14.2, *)) {
+      _pipController.canStartPictureInPictureAutomaticallyFromInline =
+          canStartPictureInPictureAutomaticallyFromInline;
+    }
+    _pipController.delegate = self;
+  }
```

Comment:
^ This too

Reply:
This feature is only supported from 14.2 so It will not show the pip when closing the app.
starting from 14.2 depending on the `canStartPictureInPictureAutomaticallyFromInline` it will got o pip when the app goes to background if `canStartPictureInPictureAutomaticallyFromInline` is true. otherwise it will just close the app. as it does on below 14.2